axelrod/strategies/worse_and_worse.py

```diff
@@ -0,0 +1,59 @@
+from axelrod import Actions, Player, random_choice
+
+C, D = Actions.C, Actions.D
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / 1000'. Therefore
+    it is more and more likely to defect as the round goes on.
+
+    Source code available at the download tab of [PRISON1998]
+
+
+    Names:
+        - worse_and_worse: [PRISON1998].
```

Comment:
You need the `_` at the end of the `[PRISON1998]`. So this should be `[PRISON1998]_` same for the one above (line 10).

Also can you change it to be:

```
- Worse and Worse: [PRISON1998]_
```